stories/card.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+import Readme from './Readme'
+import readmeCard from '../src/components/card/readme.md'
+import readmeHeading from '../src/components/card-heading/readme.md'
+
+export default {
+  title: 'Card',
+}
+
+const style = {
```

Comment:
Because Card also has styling for margin, this styling should be applied to a container instead of the card element itself for a more accurate preview